package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@identity.com/dsr",
-  "version": "1.0.23",
+  "version": "3.0.0",
```

Comment:
Dunno why I skipped v2... considering it's already release as v3-beta, probably best to continue